OPEN-32, OPEN-43: Improve services GUI

delrueba opened the pull request:
OPEN-32 & OPEN-43

Main issues: updated table layout & provided 'deactivate' button.
Also moved search field and buttons to improve ux and match other pages.

Jelle-S commented on resources/assets/js/page/PageHome.vue:
```diff
@@ -1,31 +1,36 @@
+        </form>
 
-      <button type="button" class="btn btn-success" @click="newUser" v-if="route.tab == 'users'">Gebruiker uitnodigen</button>
+      <!-- <button type="button" class="btn btn-link btn-disabled" :class="{active: route.tab=='admin'}" @click="route.tab='admin'" disabled>Toon administrators</button> -->
```
Remove this comment.

Jelle-S commented on resources/assets/js/page/PageHome.vue:
```diff
@@ -1,31 +1,36 @@
+        <!--<button type="button" class="btn btn-default" :class="{ 'btn-warning': draft }" @click="draft = true">Activeer diensten</button>-->
+      <!--</div>-->
+
+      </div>
```
Remove these comments.

Jelle-S commented on resources/assets/sass/app.scss:
```diff
@@ -24,17 +24,26 @@ body {
 .container-breadcrumb {
   margin-bottom: 0;
-  border-bottom: 1px solid #ccc;
+  //border-bottom: 1px solid #ccc;
```
Remove this comment.

Jelle-S commented on resources/assets/sass/app.scss:
```diff
@@ -24,17 +24,26 @@ body {
 .breadcrumb > .active > a {
   color: #777;
-  text-decoration: underline;
+  //text-decoration: underline;
```
Remove this comment.

Jelle-S commented on resources/assets/sass/app.scss:
```diff
@@ -55,15 +64,20 @@ body {
-  border-top-width: 1px;
+//.table > tbody + tbody {
+//  border-top-width: 1px;
+//}
```
Remove this comment.

Jelle-S commented on resources/assets/sass/app.scss:
```diff
@@ -73,17 +87,41 @@ body {
 .table {
   margin-top: 15px;
-  border-top: 1px solid #999;
+  //border-top: 1px solid #999;
```
Remove this comment.

Jelle-S commented on resources/assets/sass/app.scss:
```diff
@@ -73,17 +87,41 @@ body {
+  }
   >thead>tr>th {
-    border-bottom: 1px solid #999;
+    //border-bottom: 1px solid #999;
```
Remove this comment.

Jelle-S commented on resources/assets/sass/app.scss:
```diff
@@ -232,7 +270,7 @@ h1+h3 {
   bottom: 0;
   background: white;
-  border-top: 1px solid #ccc;
+  //border-top: 1px solid #ccc;
```
Remove this comment.

Jelle-S commented on resources/assets/sass/app.scss:
```diff
@@ -73,17 +87,41 @@ body {
+  border-collapse:collapse;
+  margin-bottom: 80px;
+
+
```
Only one newline needed.

Jelle-S commented on resources/assets/sass/app.scss:
```diff
@@ -55,15 +64,20 @@ body {
   padding-top: 4px !important;
   padding-bottom: 4px !important;
+  width: 1px;
+
```
Remove newline before closing brace.

delrueba commented:
## Style changes
Based on the mockups provided with the tickets. 

### General
- colors (~styleguide)
- search field: positioned near table
- pagination: match width of table
- extra indentation for sortable table cells
- updated breadcrumbs layout

### No active services
- changed button layout to match channels page (last image)

![afbeelding](https://user-images.githubusercontent.com/31842807/30796476-86a39b3c-a1d2-11e7-929c-deba8db72faf.png)

### Activate services
- replaced 'empty checkbox' with textual button

![afbeelding](https://user-images.githubusercontent.com/31842807/30796490-9a142bb4-a1d2-11e7-92fe-b205e97292af.png)

### Active services
- added 'deactivate' option

![afbeelding](https://user-images.githubusercontent.com/31842807/30796514-ac234b1e-a1d2-11e7-876e-97e279827f06.png)

### Channels
- for comparison only
![afbeelding](https://user-images.githubusercontent.com/31842807/30796527-bddf8d72-a1d2-11e7-999f-fe3e9dcf473e.png)
